README.md

```diff
@@ -0,0 +1,27 @@
+# go-retry
+
+![](https://github.com/vimeo/go-retry/workflows/Go/badge.svg)
+![](https://godoc.org/github.com/vimeo/go-retry?status.svg)
```

Comment:
Hmm, I'm not sure this will link the way we want.
https://godoc.org/github.com/vimeo/go-retry?tools gives me this snippet:
`[![GoDoc](https://godoc.org/github.com/vimeo/go-retry?status.svg)](https://godoc.org/github.com/vimeo/go-retry)`

Reply:
Right -- that's just giving me the badge without the link... :-(